torchtext/models/multiheadattention.py

```diff
@@ -0,0 +1,236 @@
+import torch
+from torch._jit_internal import Tuple, Optional
+
+
+Tensor = torch.Tensor
```

Comment:
nit: I would have done instead
```python
from torch import Tensor
```

but this doesn't make a difference, just stylistic